app/controllers/sac_cas/person/history_controller.rb

```diff
@@ -6,6 +6,12 @@
 #  https://github.com/hitobito/hitobito_sac_cas
 
 module SacCas::Person::HistoryController
+  extend ActiveSupport::Concern
+
+  def index
+    @external_trainings = ExternalTraining.where(person_id: @person.id).list
```

Comment:
Hier gibts imho noch n+1 queries, braucht noch `event_kind: translations` für die Anzeige und `:person`  für die Ability 

auf `feature/332-add-training-days-to-person-history` sind sonst alle anderen n+1 imho schon bereinigt

Reply:
Hmm `:person` brauchts hier nicht.
Die einzige Permission die hier abgefragt wird ist `can?(:create, @person.external_trainings.build)` und die Person hat es ja dann schon im Memory. Bullet stimmt mir jedenfalls zu :thinking: